vm/src/stdlib/cmath.rs

```diff
@@ -73,6 +73,12 @@ mod cmath {
         result_or_overflow(z, z.exp(), vm)
     }
 
+    //caluclates square Root of the complex number
```

Comment:
We don't need this comment. The method name and module obviously show what it is. Instead of it, we need doc-comment about this function to add document.
Doc-comment starts with 3 slashes like `///`. The contents can be found from CPython. (try cmath.sqrt.__doc__ in python)

Reply:
We want actual python document, not the original version.

Reply:
You mean just this one function?
Thankyou for being so helpful!